test/NITTA/Model/ProcessorUnits/Tests/DSL.hs

```diff
@@ -442,7 +463,7 @@ instance (Var v, Val x, BreakLoopProblem u v x) => Refactor u (BreakLoop v x) wh
     refactor ref = refactor' ref breakLoopOptions breakLoopDecision
 
 mkBreakLoop :: (Var v, Val x) => x -> v -> [v] -> Statement u v x (BreakLoop v x)
-mkBreakLoop x i o = return $ BreakLoop x (S.fromList o) i
+mkBreakLoop x v o = return $ BreakLoop x (S.fromList o) v
```

Comment:
Why? `i` means `input`...

Reply:
i renamed to input, o to output because i is imported from `Data.String.Interpolate`